src/XrdSys/XrdSysFD.hh

```diff
@@ -151,6 +151,10 @@ inline int  XrdSysFD_Socketpair(int domain, int type, int protocol, int sfd[2])
                  }
 #endif
 
+// openat is part of POSIX.1-2008; in Linux, BSD, and Solaris
```

Comment:
You mentioned in the pull comments that you used SysFD where possible which implies there are places which would have been good to use SysFD but were omitted allowing for a possible FD leak. Is that really the case or did this addition fix it?

Reply:
Oh - that was just imprecise english.

Unless I missed something, the remaining "open"-like calls are really just things like "fdopen" (that convert a file descriptor into a handle).

If you see any places where I open directly in this patchset, will be happy to fix.